src/components/VHeader/VHeader.vue

```diff
@@ -163,7 +161,7 @@ const VHeader = defineComponent({
     })
 
     watch(
-      () => store.state.search.query.q,
+      () => searchStore.query.q,
       (newSearchTerm) => {
         if (newSearchTerm !== localSearchTerm.value) {
           localSearchTerm.value = newSearchTerm
```

Comment:
What is this watcher meant to accomplish? How would the search store query change without the search term in the text box changing? :thinking:

Reply:
Removed :)